src/pages/resources/reference/push-notifications/templates/tabs/basic.md

```diff
@@ -91,6 +91,94 @@ Below is a sample of what a payload might look like for a basic notification:
 }
 ```
 
-<Variant platform="apns" template="basic" repeat="1"/>
+## Usage recommendations
 
-### Coming soon!
+The tables below contain guidelines for your push notification content. **These recommendations help your text display reliably across multiple devices.** These values are guidelines only - it is still recommended to test a notification prior to sending it.
+
+### Image specifications
+
+| **Aspect ratios** | **Image size range** | **Supported file types** |
+| :-------- | -----------: | ------: |
+| 1:1, 3:2, 5:4, 4:3 | < 1 MB | PNG, JPG, WebP |
+
+### Text guidelines
+
+| **Type** |**Recommended max<br /># of characters** | **Supports multiple<br />lines?** |
```

Comment:
Is there a reason for these brs here? I don't think table headers support brs.

Reply:
normal markdown calculates column width differently - the brs make this table look better on (e.g.) github.

i'll remove them from these docs though as they have no impact - all columns are assigned equal width :(